lib/text/ttml_text_parser.js

```diff
@@ -403,19 +417,37 @@ shaka.text.TtmlTextParser.parseCueRegion_ = function(regionElement, styles) {
 
   let results = null;
   let percentage = null;
+  let globalWidth = null;
+  let globalHeight = null;
+  if (globalExtent) {
+    percentage = TtmlTextParser.percentValues_.exec(globalExtent);
```

Comment:
Moving things around a little, `globalWidth` and `globalHeight` can both become const.

```
...

if (globalExtent) {
  percentage = ...
  results = ...
}

const globalWidth = results ? Number(results[1]) : null;
const globalHeight = results ? Number(results[2]) : null;

...

```